tests/unit/amo/components/TestHeroRecommendation.js

```diff
@@ -284,9 +285,10 @@ describe(__filename, () => {
       const shelfData = createShelfData({ addon: { ...fakeAddon, slug } });
 
       const root = render({ shelfData });
+      const addonURL = getAddonURL(slug);
```

Comment:
As above, we don't really need this `const`. Line 291 could just be:
`addParamsToHeroURL({ urlString: getAddonURL(slug) }),`.